torch/quantization/fx/_equalize.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, dtype=torch.quint8, qscheme=torch.per_tensor_affine,
                                                   quant_max=quant_max,
                                                   factory_kwargs=factory_kwargs)
 
-        self.equalization_scale = torch.empty(0)
+        self.equalization_scale = torch.tensor(1)
```

Comment:
is it possible that equalization_scale is actually 1?

Reply:
It could be possible that the equalization_scale is equal to 1 -- when the input column dimension is all the same number. I think the only issue with this is when scaling the next input quantization observer since we would currently use the default min/max values, but I have addressed this issue in the latest commit.